models/DataObject/ClassDefinition/Data/Time.php

```diff
@@ -143,7 +143,7 @@ public function isDiffChangeAllowed($object, $params = [])
      */
     public function isEmpty($data)
     {
-        return strlen($data) !== 5;
+        return !is_string($data) || strlen($data) !== 5;
```

Comment:
```suggestion
        return !is_string($data) || !preg_match('/^(2[0-3]|[01][0-9]):[0-5][0-9]$/', $data);
```
Maybe we should also here make the preg_math and not only the length